### PR TITLE
Release 103/50 bacteria

### PR DESCRIPTION
**Pull request by manuelcarbajo**
Fixes 2 broken links in bacteria index page:
- Ensembl API site
- Uniprot blog link

### CODE REVIEW

File: bacteria/access.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 - *Search for a gene* - type the name of a gene or other identifier into the search box above
 - *Find a genome* - click in the 'browse a genome' box above and start typing your genome name to find matching genomes
 - [View full list of all Ensembl Bacteria species](/species.html)
-- [Access Ensembl Bacteria programmatically](http://ensemblgenomes.org/info/access/eg_api)
+- [Access Ensembl Bacteria programmatically](http://bacteria.ensembl.org/info/data/api.html)
```
Comment (azangru) [inline bacteria/access.md:4]: shouldn't we encourage the use of https when providing links to our sites?

Comment (ens-ap5) [reply bacteria/access.md:4]: Sorry, that was my/Firefox's fault - I went to the bacteria website to get the correct link and just copied the URL from the browser address bar. @manuelcarbajo could you change it to just /info/data/api.html, please? It doesn't really need the full address for a link to local documentation.

Comment (manuelcarbajo) [reply bacteria/access.md]: Thank you for your comments @azangru and @ens-ap5 . I've added https to Uniprot and made the API link local

**Comment by imransl** [comment]
@manuelcarbajo Do we still need to get this PR merged for 102?

**Comment by manuelcarbajo** [comment]
yes please @imransl , I added a commit including the suggestions by @azangru and @ens-ap5

**Comment by imransl** [comment]
I've updated the title of this PR as it is for 103/50

**Pull request merged**